src/scala/com/github/johnynek/bazel_deps/Target.scala

```diff
@@ -75,6 +75,30 @@ case class Target(
   licenses: Set[String] = Set.empty,
   generateNeverlink: Boolean = false) {
 
+  def listStringEncoding(separator: String): List[String] = {
```

Comment:
can you comment the encoding here (L, B, etc..)

Reply:
👍